packages/base/src/StaticArea.js

```diff
@@ -0,0 +1,97 @@
+let staticAreaIndex = 1;
+class StaticAreaItem extends HTMLElement {
+	constructor() {
+		super();
+	}
+}
+class StaticAreaElement extends HTMLElement {
+	constructor() {
+		super();
+	}
+}
+
+class StaticArea {
+	constructor(_staticAreaTemplate, _staticAreaStyles, _renderMethod) {
+		this.staticAreaTemplate = _staticAreaTemplate;
+		this.staticAreaStyles = _staticAreaStyles;
+		this.render = _renderMethod;
+	}
+
+	/**
+	 * @protected
+	 * Creates static area as first child of the body element or returns the existing one.
+	 */
+	static _getInstance() {
+		let staticArea = document.querySelector("ui5-static-area");
+
+		if (staticArea) {
+			return staticArea;
+		}
+
+		// Create static area if it is not present
+		const bodyElement = document.body;
+		customElements.define("ui5-static-area", StaticAreaElement);
```

Comment:
Move the define to the end of the file before the export. I know it might not be needed sometimes but I think it's fine.